network-api/networkapi/wagtailcustomization/static/wagtailadmin/css/custom-fix.css

```diff
@@ -0,0 +1,42 @@
+/*
+  To address bug from Wagtail v3
+  See context: https://github.com/mozilla/foundation.mozilla.org/issues/10192
+*/
+body#wagtail
+  .sidebar-panel
+  .c-page-explorer__see-more
+  svg.icon.icon-arrow-right {
+  width: 20px;
+  height: 20px;
+  position: relative;
+  top: 5px;
+}
+
```

Comment:
Hi @mmmavis!

This is looking great to me so far, after testing, I can confirm that your code changes fix both the issues mentioned in the ticket.

One thing that I did note and wanted to bring up, was that on screen sizes larger than M, it appears that the "see more" box is taller than it needs to be. According to the inspector, it is set to 50px in height (I am assuming to make room for the larger than usual SVG): 


<img width="1919" alt="Screenshot 2023-02-24 at 11 46 47 AM" src="https://user-images.githubusercontent.com/18314510/221276832-71fde8f6-11c0-47fc-86af-915bd72079cf.png">


I was wondering what you thought about adding something like this in order to reduce the unnecessary spacing around the text?:

```suggestion

body#wagtail .sidebar-panel .c-page-explorer__see-more {
  height: unset;
}

```

 Here is what it looks like with the suggested CSS above:


<img width="1919" alt="Screenshot 2023-02-24 at 11 43 16 AM" src="https://user-images.githubusercontent.com/18314510/221276839-46263f6f-f70b-4d5e-af28-e6841fcbf07c.png">


I have also tested it on smaller screensizes and can confirm that everything is looking as expected. 

Thanks!